build-system/tasks/e2e/describes-e2e.js

```diff
@@ -253,33 +233,28 @@ function describeEnv(factory) {
     function doTemplate(name, variant) {
       const env = Object.create(variant);
       let asyncErrorTimerId;
-      this.timeout(TIMEOUT);
-      beforeEach(async() => {
-        // Set up all fixtures.
-        for (const fixture of fixtures) {
-          await fixture.setup(env);
+      this.timeout(TEST_TIMEOUT);
+      beforeEach(async function() {
+        this.timeout(SETUP_TIMEOUT);
+        await fixture.setup(env);
```

Comment:
Since `beforeEach()` is run once per test case, does it make sense for `SETUP_TIMEOUT` to be 30s and `TEST_TIMEOUT` to be 20s? Also, does `fixture.setup` have to be run once per test?

If you can get away with running it once per suite, maybe move it to `before()`?

Reply:
That's a good idea, I can look into it and see if it speeds up the job.